tests/Data/ProgrammesDb/EntityRepository/PromotionRepository/FindActivePromotionsByPidTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
+use DateTimeImmutable;
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\PromotionRepository::<public>
+ */
+class FindActivePromotionsByPidTest extends AbstractDatabaseTest
+{
+    private $promotionRepository;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->loadFixtures(['PromotionsFixture']);
+        $this->promotionRepository = $this->getRepository('ProgrammesPagesService:Promotion');
+    }
+
+    public function testActivePromotionsAreReceived()
+    {
+        $brandPid = new Pid('b010t19z');
+        $dbPromotions = $this->promotionRepository->findActivePromotionsByPid($brandPid, new DateTimeImmutable(), 300, 0);
```

Comment:
My IDE can't find the method `findActivePromotionsByPid()`. Could you typehint `private $promotionRepository;`